src/config/index.ts

```diff
@@ -4,6 +4,7 @@ export default {
     apolloClientName: 'BrazeContentProxy',
     version: `${process.env.GIT_SHA ?? 'local'}`,
     port: 4500,
+    INVALID_API_KEY_ERROR_MESSAGE: 'Please provide a valid API key.',
```

Comment:
Should we store error strings as config values? If so, I need to copy the other two error strings 😅